apps/rush-lib/src/schemas/rush.schema.json

```diff
@@ -41,6 +41,10 @@
       "description": "The minimum folder depth for the projectFolder field.  The default value is 1, i.e. no slashes in the path name.",
       "type": "number"
     },
+    "enforceConsistentVersions": {
```

Comment:
>enforceConsistentVersions [](start = 5, length = 25)

Since this is a policy, probably we should allow `--bypass-policy` in cases where the developer wants to for `rush install` anyway because the policy isn't relevant to what they're doing.